<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #F8F8F0;
border-radius: 15px;
</string>
   </property>
   <widget class="QWidget" name="opsiPanel" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblNomor">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Soal 1 / 10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPertanyaan">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Teks pertanyaan muncul di sini</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblGambarSoal">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1</width>
         <height>700</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>Gambar Soal</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblJawaban">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Jawaban:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbA">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>A. Opsi A</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="rbB">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>B. Opsi B</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="rbC">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>C. Opsi C</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="rbD">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>D. Opsi D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblBenar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Jawaban benar: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnBack">
          <property name="styleSheet">
           <string notr="true">background-color: #AED581;
border-radius: 10px;
padding: 8px;
</string>
          </property>
          <property name="text">
           <string>Kembali</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNext">
          <property name="styleSheet">
           <string notr="true">background-color: #AED581;
border-radius: 10px;
padding: 8px;
</string>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
